src/components/Container/Container.scss

```
@@ -0,0 +1,117 @@
+@import './../../styles/fonts.scss';
+@import './../../styles/breakpoints.scss';
+@import './../../styles/alignment.scss';
+
+.container {
+  display: grid;
+  text-align: center;
+  max-width: $breakpoint-laptop-large;
+  margin: 0 16px;
+  padding: 5px;
+}
+
+// This column class could be removed entirely and the grid system would work. We could consiser adding the padding and margin to the layout classes, and controlling this in the layout classes.
+
+.column {
+  padding: 10px;
+  margin: 10px;
+}
+
+/* Layouts  */
+
+// 2 Columns
+
+.cols-2 {
+  grid-template-areas: 'column column';
+  grid-template-columns: 1fr, 1fr;
+  grid-auto-flow: column;
+  @media screen and (max-width: $breakpoint-tablet) {
+    grid-template-areas: 'column column';
+    grid-template-columns: 1fr 1fr;
+  }
+  @media screen and (max-width: $breakpoint-mobile) {
+    grid-template-areas:
+      'column'
+      'column';
+    grid-template-columns: 1fr;
+  }
+}
+
+// 3 Columns
+
+.cols-3 {
+  grid-template-areas: 'column column column';
+  grid-template-columns: repeat(3, 1fr);
+  grid-auto-flow: column;
+  @media screen and (max-width: $breakpoint-tablet) {
+    grid-template-areas: 'column column column';
+    grid-template-columns: repeat(3, 1fr);
+  }
+  @media screen and (max-width: $breakpoint-mobile) {
+    grid-template-areas:
+      'column'
+      'column'
+      'column';
+    grid-template-columns: 1fr;
+  }
+}
+
+// 4 Columns
+
+.cols-4 {
+  grid-template-areas: 'column column column column';
+  grid-template-columns: repeat(4, 1fr);
+  @media screen and (max-width: $breakpoint-tablet) {
+    grid-template-areas: 'column column';
+    grid-template-columns: 1fr 1fr;
+  }
+  @media screen and (max-width: $breakpoint-mobile) {
+    grid-template-areas:
+      'column'
+      'column'
+      'column'
+      'column';
+    grid-template-columns: 1fr;
+  }
+}
+
+// 5 Columns
+
+.cols-5 {
+  grid-template-areas: 'column column column column column';
+  grid-template-columns: repeat(5, 1fr);
+  @media screen and (max-width: $breakpoint-tablet) {
+    grid-template-areas: 'column column column column column';
+    grid-template-columns: repeat(5, 1fr);
+  }
+  @media screen and (max-width: $breakpoint-mobile) {
+    grid-template-areas:
+      'column'
+      'column'
+      'column'
+      'column'
+      'column';
+    grid-template-columns: 1fr;
+  }
+}
+
+// 6 Columns
+
+.cols-6 {
+  grid-template-areas: 'column column column column column column';
+  grid-template-columns: repeat(6, 1fr);
+  @media screen and (max-width: $breakpoint-tablet) {
+    grid-template-areas: 'column column column';
+    grid-template-columns: repeat(3, 1fr);
+  }
+  @media screen and (max-width: $breakpoint-mobile) {
+    grid-template-areas:
+      'column'
+      'column'
+      'column'
+      'column'
+      'column'
+      'column';
+    grid-template-columns: 1fr;
+  }
+}
```

The columns seem repetitive. Since most of the attribute and values are similar, would it be possible to implement a SASS for loop to generate these columns? 
```suggestion
$total-columns: 6;

@for $i from 1 through $total-columns {
  .col-#{$i} {
    grid-template-columns: repeat(#{$i}, 1fr);
    grid-auto-flow: column;

    @media screen and (max-width: $breakpoint-tablet) {
      grid-template-columns: repeat(#{$i}, 1fr);
    }

    @media screen and (max-width: $breakpoint-mobile) {
      grid-template-columns: 1fr;
    }
  }
}
```

---

i removed grid-template-areas in the suggestion, if it still needs to be included you can use the repeat function in the suggestions

---

@mattlaguardia & @mcheng09 , I've added the SASS loop, and added a calc function that will divide the total amount of columns by 2 for the tablet breakpoint, and changed the `grid-auto-flow:` from `column` to `row`.